### DISCUSSION TITLE
How can I see the LazyVim keymaps for Telescope?

### DISCUSSION THREAD

**Discussion by barrelltech** [LazyVim/LazyVim#4759] [Q&A]
Long time neovim user, just trying LazyVim.

Telescope has some default keybindings normally - `<C-/>` to see help, `<C-x>` to open in a vertical split, etc.

However these do not work in the Telescope for LazyVim. `<C-x>` adds letters to all the results, and `<C-s>` opens the result in a vertical split.

I just happen to guess the `<C-s>` split, I have no idea how to see what the telescope mappings are for LazyVim. I've poured over the documentation and all the files I can think to check and I see no remappings in Telescope, yet there are very clearly non-vanilla-telescope keycaps.

How can I see the LazyVim keycaps available while in a Telescope search?

**Comment by zvrkSam** [comment]
Try `<leader>sk`

**Comment by dpetka2001** [comment]
`<C-/>` works perfectly fine for me in Telescope. From what you're describing it seems that you use `FzfLua` instead of `Telescope`. Are you sure you didn't enable the `FzfLua` Extra and you're using that instead of Telescope?

PS: In case of `FzfLua` you can press `F1` to show help about the mappings it uses.
